AccessCheckoutSDK/AccessCheckoutSDKTests/mock/MockCVVOnlyDelegate.swift

```diff
@@ -0,0 +1,96 @@
+// MARK: - Mocks generated from file: AccessCheckoutSDK/cvvOnly/CVVOnlyDelegate.swift at 2020-03-25 15:29:55 +0000
```

Comment:
I think there's an option in the mock generation library to prevent it from adding this line. I'll have a look at that cause that line should not be there really

Reply:
Done.